package.json

```diff
@@ -46,6 +20,7 @@
   "engine": {
     "node": ">=18.16"
   },
+  "packageManager": "npm@9.5.0",
```

Comment:
I would not include it there, we get our package manager requirement based on node version above and this would be another arbitrary version bump to not forget about when updating

Reply:
Will remove it

Reply:
removed